src/amo/components/Errors/NotAuthorized/index.js

```diff
@@ -32,25 +33,21 @@ export class NotAuthorizedBase extends React.Component<Props> {
     // TODO: Offer a sign in link/button inside the error page.
     /* eslint-disable react/no-danger */
     return (
-      <NestedStatus code={401}>
```

Comment:
All of these `NestedStatus` components should remain in the low-level error component, not the page component.

For example, trigger a 404 on the search page http://localhost:3000/en-US/firefox/search/?platform=mac&q=yes&page=9999

The status code is a 200 which is incorrect.